x-pack/plugins/ml/server/routes/annotations.ts

```diff
@@ -130,6 +136,9 @@ export function annotationRoutes(
       validate: {
         params: deleteAnnotationSchema,
       },
+      options: {
+        tags: ['access:ml:canCreateJob'],
```

Comment:
As above. I think a user should be allowed to delete an annotation, even if they can't create jobs. Maybe this should be `access:ml:canGetJobs` or we could create new capability(s) for adding / deleting annotations?